src/drafts/MarkdownEditor/_useListEditing.ts

```diff
@@ -29,7 +29,7 @@ const calculateNextListItemStarter = ({leadingWhitespace = '', delimeter, taskBo
  *  3. Task box (optional)
  *  4. Everything following
  */
-export const listItemRegex = /^(\s*)([*-]|(\d+)\.)\s(?:(\[[\sx]\])\s)?(.*)/i
+export const listItemRegex = /^(\s*)([*-]|(\d+)\.)(\s{1,2})(?:(\[[\sx]\])\s)?(.*)/i
```

Comment:
Are tab characters valid here? Maybe more correct would be space only (` `). Not sure though. 

```suggestion
export const listItemRegex = /^(\s*)([*-]|(\d+)\.)( {1,2})(?:(\[[\sx]\])\s)?(.*)/i
```

Reply:
`\s` is identifying tabs as well, i just checked